installing/installing_aws/installing-aws-expanding-a-cluster-with-on-premise-bare-metal-nodes.adoc

```diff
@@ -21,4 +21,10 @@ include::modules/installation-aws_con_connecting-the-vpc-to-the-on-premise-netwo
 
 include::modules/installation-aws_proc_creating-firewall-rules-for-port-6183.adoc[leveloffset=+1]
 
-After you have the networking configured, you can proceed with xref:../installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc#ipi-install-expanding-the-cluster[expanding the cluster].
\ No newline at end of file
+After you have the networking configured, you can proceed with expanding the cluster.
+
+include::modules/configuring-a-provisioning-resource-to-scale-the-cluster.adoc[leveloffset=+1]
+include::modules/adding-new-hosts-in-the-cluster-with-the-bmo.adoc[leveloffset=+1]
+
+
+
```

Comment:
Remove empty lines (I think this might be recommended due to publishing tools having issues with them)